packages/plugins/plugin-assistant/src/functions/load.ts

```diff
@@ -0,0 +1,44 @@
+//
+// Copyright 2025 DXOS.org
+//
+
+import { Effect, Option, Schema, pipe } from 'effect';
+
+import { ArtifactId } from '@dxos/assistant';
+import { Obj } from '@dxos/echo';
+import { DatabaseService, defineFunction } from '@dxos/functions';
+import { log } from '@dxos/log';
+
+// TODO(burdon): Common error handling.
+// TODO(burdon): Factor out to space plugin.
+export default defineFunction({
```

Comment:
i was thinking that we'd move this to the space plugin, but i think it's good to have it here for the time being.